app/views/maintenance_tasks/runs/_info.html.erb

```diff
@@ -13,14 +13,27 @@
   <div class="block">
     <%= link_to("Download CSV", csv_file_download_path(run)) %>
   </div>
+  <hr/>
 <% end %>
 <% if run.arguments.present? %>
-  <div class="block">
-    <h5>Arguments:</h5>
-    <pre>
-      <% run.arguments.each do |key, value| %>
-        <%= "#{key}: #{value}" %>
-      <% end %>
-    </pre>
+<div class="table-container">
+  <h6 class="title is-6">Arguments:</h6>
+    <table class="table">
+      <tbody>
+        <% run.arguments.each do |key, value| %>
+          <tr>
+            <td><var><%= key %></var></td>
```

Comment:
Not a fan of the `<var>`. What I really want is monospace, so that argument names look like code, but `<code>` is a bit too intense for the label and in the table. I pushed a commit to use a class to add styling instead.

Reply:
Nice! Yeah sorry, forgot to mention that I went with `<var>` because `<tt> is actually deprecated. The styling looks nice this way 👍